kakao/src/main/kotlin/com/agoda/kakao/delegates/WebInteractionDelegate.kt

```diff
@@ -0,0 +1,45 @@
+package com.agoda.kakao.delegates
+
+import android.support.annotation.CheckResult
+import android.support.test.espresso.web.assertion.WebAssertion
+import android.support.test.espresso.web.model.Atom
+import android.support.test.espresso.web.model.ElementReference
+import android.support.test.espresso.web.sugar.Web
+import com.agoda.kakao.configurator.KakaoConfigurator
+import org.hamcrest.Matcher
+import javax.annotation.CheckReturnValue
+
+/**
+ * An interface duplicating the used in Kakao part of web interaction interface and providing original Espresso *Interactions.
+ * If you want to extend the base web interaction behavior, you should implement this interface, and specify it's
+ * factory in [com.kaspersky.klkakao.configurator.KakaoConfigurator]
```

Comment:
Same

Reply:
As I mentioned above in my first message at this PR I would update docs later